Sources/AppCoinsSDK/Domain/Entities/ClientWallet.swift

```diff
@@ -51,12 +51,12 @@ internal class ClientWallet: Wallet, Codable {
         }
     }
     
-    internal func getBalance(currency: Coin, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
+    internal func getBalance(currency: String, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
         if let address = self.address {
             transactionService.getBalance(wa: address) { result in
```

Comment:
TransactionService.getBalance is using a deprecated request. We should change it to this new endpoint: `https://apichain.catappult.io/transaction/1.20230807/wallet/\(wallet_address)/info?currency=\(currency)`, where the balance value in the user currency will be on the parameter `appc_c_fiat_balance`.

Reply:
Done